lib/code_corps/policy/project_user.ex

```diff
@@ -7,17 +7,18 @@ defmodule CodeCorps.Policy.ProjectUser do
   alias CodeCorps.{ProjectUser, Repo, User}
   alias Ecto.Changeset
 
-  @spec create?(User.t, Ecto.Changeset.t) :: boolean
-  def create?(%User{id: id}, %Changeset{changes: %{user_id: user_id}}), do:  id == user_id
-  def create?(%User{}, %Changeset{}), do: false
+  @spec create?(User.t, map) :: boolean
+  def create?(%User{id: user_id}, %{"user_id" => author_id})
+    when user_id == author_id and not is_nil(user_id), do: true
+  def create?(%User{}, %{}), do: false
 
-  @spec update?(User.t, Ecto.Changeset.t) :: boolean
-  def update?(%User{} = user, %Changeset{data: %ProjectUser{} = record} = changeset) do
-    user_membership = record |> get_project_membership(user)
+  @spec update?(User.t, ProjectUser.t, map) :: boolean
+  def update?(%User{} = user, %ProjectUser{} = existing_record, params) do
```

Comment:
The change requires to take 3 params. This part of code caused a lot of change in the corresponding, `project_user_test.ex` file